experimental/PropertyDDS/packages/property-inspector-table/webpack.appfw.base.js

```diff
@@ -51,9 +51,7 @@ module.exports.TSConfig = function(fileType, generateDeclarations, distTypesPath
         rules: [
           {
```

Comment:
Probably we should rename this file

Reply:
good point. I'll do that in the other PR.